tap_sftp/sync.py

```diff
@@ -32,18 +32,24 @@ def sync_stream(config, state, stream, sftp_client):
         table_spec["search_pattern"],
         modified_since
     )
-    # sftp_client.close()
```

Comment:
This comment was leftover from the previous author, see this commit: https://github.com/Shopify/tap-sftp-xml-support/commit/f42a4bb2b460e1143478b4e2ea2f47ab052a0186

The client is closed [here](https://github.com/Shopify/tap-sftp-xml-support/blob/3113bca570e81cfca9a5993d8f6420fa260c88f9/tap_sftp/tap.py#L74).